lib/cc/engine/analyzers/engine_config.rb

```diff
@@ -14,6 +14,10 @@ def languages
           config.fetch("languages", {})
         end
 
+        def concurrency
```

Comment:
I'd cap it at 2 for now. Later, we'll probably make this externally configurable by a build runner so that we can set the max to other values.